docs/content/Configuration/Config.md

```diff
@@ -262,20 +262,25 @@ module.exports = {
 };
 ```
 
-### queryTransformer
+### queryRewrite
+
+<!-- prettier-ignore-start -->
+[[warning | Note]]
+| In previous versions of Cube.js, this was called `queryTransformer`.
```

Comment:
@ovr It is great that you have added this note, thank you! @hassankhan do you think we need to mention something around that `queryTransformer` name still supported and that we kept the backward compatibility?